### PR TITLE
188497532 graph map filter formula editor

### PR DESCRIPTION
**Pull request by eireland**
Changes the graph filter formula editor to use the more complete formula editor modal.

Adds Edit Filter Formula to the map inspector panel, which also uses the more complete formula editor modal. Note that the actual filtering has not been implemented in the map.

### CODE REVIEW

**Review by tealefristoe — APPROVED** [review]
Looks good 👍 

Only big thing to do is remove the outdated `edit-filter-formula-modal.tsx`.

One other thing you might consider doing is making `dataConfig.filterFormula.addOrEditFormalString` a view on the model so you don't repeat the same code in both `hide-show-menu-list.tsx` files.

**Review by tealefristoe — CHANGES_REQUESTED** [review]
Actually it looks like a cypress test is failing after this change. You should apply the `run regression` label to PRs to make sure all tests pass before requesting a review.

**Comment by kswenson** [comment]
> One other thing you might consider doing is making `dataConfig.filterFormula.addOrEditFormalString` a view on the model so you don't repeat the same code in both `hide-show-menu-list.tsx` files.

Localizable strings for menu items isn't really a model-level concern, but sharing the code some other way, e.g. a custom hook that might allow some other part of the implementation to be shared might make sense.

File: v3/src/components/graph/components/inspector-panel/hide-show-menu-list.tsx [hunk 1]
```diff
@@ -6,7 +6,8 @@ import { ITileModel } from "../../../../models/tiles/tile-model"
 import { isGraphContentModel } from "../../models/graph-content-model"
 import { t } from "../../../../utilities/translation/translate"
 import { logMessageWithReplacement } from "../../../../lib/log-message"
-import { EditFilterFormulaModal } from "../../../common/edit-filter-formula-modal"
```
[the remark this answers is not in the document]
Comment (eireland) [reply v3/src/components/graph/components/inspector-panel/hide-show-menu-list.tsx:9]: EditFilterFormulaModal is still being used in the case table/card, and will be removed when this [story](https://www.pivotaltracker.com/story/show/188497528) is worked on.

**Review by kswenson — APPROVED** [review]
👍 LGTM

**Review by tealefristoe — APPROVED** [review]
Looks good 👍 

Sorry for the misunderstanding with `EditFilterFormulaModal` earlier. I was confused because it's used in two files called `hide-show-menu-list.tsx`, and this PR modifies two files called `hide-show-menu-list.tsx`, but they weren't the same two... there are actually three files with that same name 😕

**Pull request merged**